snapcraft.yaml

```diff
@@ -1544,21 +1554,24 @@ parts:
       set -eux
       craftctl default
```

Comment:
Can we add a comment here explaining what we're undertaking in this override-build?